src/app/collections/collections.tsx

```diff
@@ -42,6 +45,7 @@ export default class Collections extends React.Component<Props, State> {
     });
 
     const profileResponse = await getKiosks(this.props.account);
+    await fetchRatings(this.props.dimDestinyTrackerService, undefined, undefined, profileResponse, defs);
```

Comment:
Hmm, it looks like this blocks loading until the ratings are available. Could we fetch them async and then set the state after, so we can render items first then update them with ratings?